metasv/generate_final_vcf.py

```diff
@@ -144,8 +144,14 @@ def filter_confused_INS_calls(nonfilterd_bed, filterd_bed, wiggle=20):
     bedtool_bp_nonINS=pybedtools.BedTool(nonINS_bp_intervals) 
     bad_INS=bedtool_INS.window(bedtool_bp_nonINS,w=wiggle)
 
-    bedtool_filtered=bedtool_INS.window(bad_INS,w=wiggle,v=True).saveas(filterd_bed)
-    bedtool_filtered =bedtool_filtered.cat(bedtool_others,postmerge=False).sort().saveas(filterd_bed)
+    bedtool_filtered=bedtool_INS.window(bad_INS,w=wiggle,v=True).saveas()
```

Comment:
It was. An empty saveas() asks pybedtools save it as a temporary file. I thought it was a bit clearer here since we save the final filterd_bed output in the next step. It's no big deal to switch it back the old way either.